### ISSUE TITLE
No warning when diary entry is not saved due to lack of disk space

### ISSUE THREAD

**Issue by spaceChRiS** [billthefarmer/diary#86]
Even if a diary entry cannot be saved due to lack of space, no warning is issued and the entry shows up in markdown view. Visiting the day again or closing the application removes the entry. I lost several entries due to this behavior. If space is low before, it should not even be possible to make an entry, and if space gets low during editing, a warning should be issued.

**Comment by billthefarmer** [comment]
I am surprised you don't get a warning from android that space is low. There are a lot of search results online on how to deal with android low space warnings. I have added an alert if an entry fails to save, but I consider it out of scope for a diary app to monitor storage space.

**Comment by spaceChRiS** [comment]
I get a warning, but that starts somewhere around 500 MB free space. Since my phone is pretty old, I am running out of space all the time. Therefore I tend to ignore the warning until I find time to tidy up. Especially, since the automatic processes that fill up my space seem not to go below a magic limit around 20 MB, which is still enough for some quick diary entries. However, last week was different as it seems that the space was entirely gone, or that diary was not able to save for some other reason. However, after removing some files, it saved again, so I concluded that free space was the reason.

All of this would be no problem, but diary did not tell that it was not able to save. The text was showing up as usual, but was gone after opening the program again. Also, going to another day and then back made the text go. And, somehow, the next/last entry buttons were not working correct anymore as well.

So, tl;dr, thanks for introducing an alert, that's great.
